CHANGES.md

```diff
@@ -6,6 +6,9 @@
 * Compiler: separate compilation can now drops unused units when linking (similar to ocamlc). (#1378)
   Feature is disabled by default while dune rules are being fixed. Enable with --enable=auto-link.
 * Compiler: specialize string to js-string conversion for all valid utf8 strings (previously just ascii)
+* Compiler: JavaScript files generated by `js_of_ocaml` are now UTF-8 encoded.
+  You need to make sure your web server serves them with that encoding or integrate
+  them in your HTML with `<script src="…" charset="UTF-8"></script>`
```

Comment:
Given what I know now, in the end I think this is going to create more confusion. Let's keep that 

``Compiler: JavaScript files generated by `js_of_ocaml` are now UTF-8 encoded.``

Reply:
done